apps-rendering/src/bodyElement.ts

```diff
@@ -475,6 +475,32 @@ const parse =
 				return parseAtom(element, atoms, context.docParser);
 			}
 
+			case ElementType.LIST: {
+				const listTypeData = element.listTypeData;
+
+				if (!listTypeData) {
+					return Result.err('No listTypeData on list element');
+				}
+
+				const parseTitle = (
+					title: string,
+				): Array<Result<string, BodyElement>> => {
+					const doc = context.docParser(`<h2>${title}`);
```

Comment:
do we need to close the `<h2>` tag here?

Reply:
Oops